### DISCUSSION TITLE
Can't Log in with Github on CodeMagic WebApp

### DISCUSSION THREAD

**Discussion by Anotherjez** [codemagic-ci-cd/codemagic-docs#2653] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

Log in usign GitHub button.

### Expected results

Expecting to log in...

### Actual results

It keeps getting stuck on the “Authenticating...” screen.

*Using Brave browser, tried with all shields off, tried with Firefox browser, tried to revoke Github Connection and reconnect, tried to sign in with my email, no vpn, nothing works...

![Screenshot 2024-07-26 130635](https://github.com/user-attachments/assets/e4c673ea-2dd8-45b1-a892-cdd4db623a22)

**Comment by dtrdic** [comment]
hello @Anotherjez can you please share with me the email you are using to log in. I'll remove this email from the thread as soon as I check the logs

**Reply by dtrdic** [reply]
hey @Anotherjez thank you for sharing. I deleted the message with your email for security reasons.
I have also disconnected the integration to your GitHub account, could you please try to reconnect to Codemagic once again?

**Reply by Anotherjez** [reply]
Same problem, stuck on Authenticating...

I also disconnected Codemagic from my Github account to reconnect it and it didn't work.

**Reply by dtrdic** [reply, marked as answer]
@Anotherjez can you please try like this;
-Go to https://github.com/apps/codemagic-ci-cd and click 'Configure'
- Uninstall the Codemagic CI/CD application from integrations
![Screenshot 2024-07-31 at 08 43 07](https://github.com/user-attachments/assets/2bf2afb0-a798-437b-aead-aa4647f9abb1)

- retry again to connect

**Reply by Anotherjez** [reply]
@dtrdic Thanks!! Now its working!
